packages/graphql-toolbox/src/modules/EditorView/Editor.tsx

```diff
@@ -165,14 +296,13 @@ export const Editor = ({ schema }: Props) => {
                             queryEditor={
                                 schema ? (
                                     <GraphQLQueryEditor
-                                        schema={schema}
                                         loading={loading}
-                                        mirrorRef={refForQueryEditorMirror}
-                                        executeQuery={onSubmit}
-                                        query={useStore.getState().getActiveTab().query}
-                                        onChangeQuery={(query) => {
-                                            store.updateQuery(query, useStore.getState().activeTabIndex);
-                                        }}
+                                        elementRef={elementRef}
+                                        // onChangeQuery={(query) => {
```

Comment:
Shall we remove these comments?

Reply:
:+1: https://github.com/neo4j/graphql/pull/3468/commits/04031407b06957d9abc014d383c6fa1cf830308d